### ISSUE TITLE
about dataset splits

### ISSUE THREAD

**Issue by HongGu-Jeong** [pclucas14/lidar_generation#21]
hello!! thank you for your awesome work

I have a question about dataset splits which mentioned in your paper

("We use the train/validation/test set split proposed
by [33], which yields 40 000, 80 and 700 samples for train,
validation and test sets")

i checked [33] ref. paper and find  "'city', '2011_09_26_drive_0005_sync' " is validation set, but it have 152 samples( after preprocess ).

This is different from what was mentioned. 

can you explain about these differences.

thanks for reading this !

**Comment by pclucas14** [comment]
Hi, 

I don't exacty recall how I did the splits, but from the paper is seems that I subsampled the recordings 

``` For training we subsample from 10 Hz to 3 Hz since temporally adjacent frames are nearly identical.```

So maybe this explains why there are less frames. Are you looking to rebuild the dataset I used for the paper ?

**Comment by HongGu-Jeong** [comment]
Thank you for the quick reply to my question 

Yes, I'm looking to rebuild the dataset you used for the paper

And following your previous answers in "Issues", i did it based on /kitti_data/parse_velo.py

I obtained train/validation/test set that is 40814 / 152 / 830 samples respectively.
(after preprocess)

I can't use "processed-kitti-lidar repo"  because it does not contain intensity that I want to know

but I checked that the number of [city-2011_09_26_drive_0005_sync](validation set) in both "processed-kitti-lidar repo" and reconstruction dataset which i rebuild are same.

I am a little concerned because the reconstructed dataset and the dataset mentioned in the paper are different.

**Comment by pclucas14** [comment]
Hi, 

So I checked the original training files and 40814 --> 13608 / 152 / 830 looks ok. However, I have never used the intensity value, so the dataset does not have them. I think I only downsampled the training set but not the validation or test sets. 

I am not sure what would be the best way to reconstruct the dataset with the intensity values. You would have to redownload the raw files (as done [here](https://github.com/pclucas14/prednet/blob/master/process_kitti.py)) and apply my preprocessing script to the files. It will be a pain though; I remember that for some point clouds it didn't work, and I had to manually remove them

Best of luck!
-Lucas
